packages/wonder-blocks-core/util/enumerate-scroll-ancestors.ts

```diff
@@ -46,13 +46,13 @@ class ScrollAncestorsIterator implements Iterator<Element> {
         return this;
     }
 
-    next(): IteratorResult<Element, void> {
+    next(): IteratorResult<Element> {
         if (this.done) {
-            return {done: true};
+            return {done: true, value: undefined};
```

Comment:
In retrospect should probably use `// @ts-ignore` until https://github.com/Microsoft/TypeScript/issues/11375 is addressed.